packages/frontend/app/routes/wallet-addresses.create.tsx

```diff
@@ -109,8 +109,11 @@ export async function action({ request }: ActionFunctionArgs) {
     return json({ errors }, { status: 400 })
   }
 
+  const baseUrl = removeTrailingSlash(getOpenPaymentsUrl())
+  const path = removeTrailingSlash(result.data.name)
```

Comment:
We can update the function name to be `removeTrailingAndLeadingSlash` to be clear its from both sides of the string